notebook/Solution/03 - Time Series Forecasting Using MLP.ipynb

```diff
@@ -0,0 +1,1693 @@
+      "text/plain": [
+       "<Figure size 1008x504 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax = plt.subplots()\n",
+    "fig.set_size_inches(14, 7)\n",
+    "ax.set_title(\"Loss Score against Epoch\")\n",
+    "ax.grid(b=True)\n",
+    "ax.set_xlabel(\"Epoch Number\")\n",
+    "ax.set_ylabel(\"Loss Score\")\n",
+    "\n",
+    "ax.plot(loss_score['train'], color='goldenrod', label='Training Loss')\n",
+    "ax.plot(loss_score['test'], color='green', label='Test Loss')\n",
+    "ax.legend();"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## <a name=\"evaluation\">Evaluation</a>\n",
+    "There are a few metrics that can be used to evaluate the performance of a regression model, which include:\n",
+    "* mean squared error (MSE)\n",
+    "* root mean squared error (RMSE)\n",
+    "* mean absolute error (MAE)\n",
+    "\n",
+    "We will examine the performance of the model on test set using RMSE. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The RMSE for the model is 89.80567585554041\n"
+     ]
+    }
+   ],
+   "source": [
+    "model.eval()\n",
+    "\n",
+    "predictions = []\n",
+    "count = 0\n",
+    "\n",
+    "# model predictions\n",
+    "with torch.no_grad():\n",
+    "    for i, (X, y) in enumerate(DataLoader(test_Dataset, 1)):\n",
+    "        count += 1\n",
+    "        output = model(X)\n",
+    "        predictions.append(output.item())\n",
+    "        \n",
+    "# convert list into np array and inverse transform it into proper values before feature scaling\n",
+    "predictions = np.array(predictions)\n",
+    "predictions = scaler.inverse_transform(predictions.reshape(1,-1))\n",
+    "\n",
+    "# remove beginning 3 rows of data since at least 3 data are needed to make a prediction\n",
```

Comment:
why need to remove the beginning of 3 rows? because of the window size?

Reply:
Beginning 3 rows are NA